csrankings.csv

```diff
@@ -19151,6 +19152,7 @@ Val Tannen,University of Pennsylvania,http://www.cis.upenn.edu/~val,NOSCHOLARPAG
 Valdis Berzins,Naval Postgraduate School,http://faculty.nps.edu/berzins/NPSVita_berzins.htm,NOSCHOLARPAGE
 Valentina A. M. Tamma,University of Liverpool,http://www.csc.liv.ac.uk/~valli,tYimqvsAAAAJ
 Valentina Boeva,ETH Zurich,http://boevalab.inf.ethz.ch,f196u4IAAAAJ
+Valentina Nisi,Universidade de Lisboa,https://www.valentinanisi.com/,1vA3h5IAAAAJ

```

Comment:
FWIW, this faculty member's website says "Valentina is currently Assistant Professor in Digital Interactive Media at the University of Madeira and Adjunct Faculty at the HCI Institute at Carnegie Mellon University." (I verified they are at Univ. of Lisboa via Google Scholar).